src/client/Microsoft.Identity.Client/ManagedIdentity/CsrMetadata.cs

```diff
@@ -0,0 +1,82 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using Microsoft.Identity.Client.Core;
+
+namespace Microsoft.Identity.Client.ManagedIdentity
+{
+    /// <summary>
+    /// Represents metadata required for Certificate Signing Request (CSR) operations.
+    /// </summary>
+    internal interface CsrMetadataInterface
```

Comment:
Avoid creating interfaces for data objects. They serve no purpose.

Reply:
done